packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -264,6 +264,10 @@ export type VideoMetadata = {
    * Secondary text that will be displayed under the title.
    */
   artist?: string;
+  /**
+   * The artwork of the video.
```

Comment:
```suggestion
   * The uri of the video artwork.
```

Let's make sure the users know what to use here